documentation/docs/quick-start.md

```diff
@@ -256,15 +256,15 @@ curl -X 'POST' \
 
 
 ### Starting Sample App
-All wallet SDK's come bundled with a sample application, that cover all the PRISM flows, including establishing connections, issuance, and verification flows.
+All wallet SDK's come bundled with a sample application, that cover all the Identus flows, including establishing connections, issuance, and verification flows.
 
 <Tabs>
 <TabItem value="js" label="Typescript Sample APP">
 
-1. Clone the [TypeScript SDK](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts) repository.
+1. Clone the [TypeScript SDK](https://github.com/input-output-hk/identus-edge-agent-sdk-ts) repository.
```

Comment:
Quick Start Guide is renamed by @todorkoleviohk.
Anyway, do not rename the wallet sdk repo as it is not existing yet. This is part of the actions to do after the migration to Hyperledger is done.

Reply:
fixed